src/Restaurant.js

```diff
@@ -1,17 +1,21 @@
-import React from "react";
+import React, { Component } from "react";
+import RestaurantButton from "./RestaurantButton";
+import Orders from "./Orders";
 
-const Restaurant = () => {
-  const pizzas = 0;
-  return (
-    <div>
-      <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
-      </ul>
-    </div>
-  );
-};
-
-export default Restaurant;
+class Restaurant extends Component {
+  render() {
+    return (
+      <div>
+        <ul>
+          <li>
+            <Orders orderType="Salad" />
+          </li>
+          <li>
+            <Orders orderType="Pizza" />
```

Comment:
Can you see how it is useful to make components reusable (but configurable with props)?